src/integration/integration_scheduler_scalingengine_test.go

```diff
@@ -101,6 +104,107 @@ var _ = Describe("Integration_Scheduler_ScalingEngine", func() {
 		})
 	})
 
+	Describe("Synchronized Schedule", func() {
+		Context("when the app's policy has been updated", func() {
+			BeforeEach(func() {
+				policyStr = string(setPolicyDateTime(readPolicyFromFile("fakePolicyWithSpecificDateSchedule.json")))
+				resp, err := createSchedule(testAppId, testGuid, policyStr)
+				checkResponseIsEmpty(resp, err, http.StatusOK)
+
+				Eventually(func() bool {
+					return activeScheduleExists(testAppId)
+				}, 2*time.Minute, 5*time.Second).Should(BeTrue())
+
+				insertPolicyDirectly(testAppId, policyStr, anouterGuid)
+
+			})
+			It("updates the schedules", func() {
+				resp, err := synchronizeSchedule()
+				Expect(err).NotTo(HaveOccurred())
+				Expect(resp.StatusCode).To(Equal(http.StatusOK))
+
+				Eventually(func() bool {
+					return activeScheduleExists(testAppId)
+				}, 2*time.Minute, 5*time.Second).Should(BeTrue())
+
+			})
+		})
+
+		Context("when the app's policy has been deleted", func() {
+			BeforeEach(func() {
+				policyStr = string(setPolicyDateTime(readPolicyFromFile("fakePolicyWithSpecificDateSchedule.json")))
+				resp, err := createSchedule(testAppId, testGuid, policyStr)
+				checkResponseIsEmpty(resp, err, http.StatusOK)
+
+				Eventually(func() bool {
+					return activeScheduleExists(testAppId)
+				}, 2*time.Minute, 5*time.Second).Should(BeTrue())
+
+				deletePolicy(testAppId)
```

Comment:
Another method call insertPolicy with the parameter of models.ScalingPolicy. Here I only need to insert with a string._